metrics/grafana/common.py

```diff
@@ -592,14 +627,27 @@ def expr_histogram_avg(
 
 
 def target(
-    expr: Union[Expr, str],
+    expr: Union[Expr, OpExpr, str],
     legend_format: Optional[str] = None,
     hide=False,
     data_source=DATASOURCE,
     interval_factor=1,  # Prefer "high" resolution
+    additional_groupby=False,
```

Comment:
```suggestion
    additional_groupby: list[str] = [ADDITIONAL_GROUPBY],
```

It looks more natural to me if it's a list of string (labels).